client/lib/resize-image-url/index.js

```diff
@@ -73,20 +73,21 @@ export default function resizeImageUrl( imageUrl, resize, height, makeSafe = tru
 		return imageUrl;
 	}
 
-	const parsedUrl = parse( imageUrl, true, true );
-	if ( ! REGEXP_VALID_PROTOCOL.test( parsedUrl.protocol ) ) {
+	const resultUrl = getUrlParts( imageUrl );
+	delete resultUrl.search;
```

Comment:
same fun fact as above; I'm sure you know this already just pointing it out

```js
const { search, ...resultUrl } = getUrlParts( imageUrl );
```

Reply:
Done, thanks!